ginga/rv/plugins/MultiDim.py

```diff
@@ -288,6 +304,7 @@ def play_axis_change():
                         self.play_max = dims[self.play_axis]
 
                     self.play_idx = self.play_indices[n - 2]
+                    self.play_last[imname] = self.play_idx
```

Comment:
Yes, I agree. I couldn't find any good API documentation for `naxispath` usage, so I used this workaround to at least get the behavior right first. I'll put it on my to-do list to incorporate your suggestions. Thank you.